src/js/variant-widget/eva-variant-widget-panel.js

```diff
@@ -343,6 +343,333 @@ EvaVariantWidgetPanel.prototype = {
+                case 'bbubalis_umdcaspurwb20':
+                    defaultRegion = 'AWWX01438720.1:1-1000000';
+                    break;
+                case 'bgrunniens_umd311':
+                    defaultRegion = 'Chr6:71000001-72000000';
+                    break;
+                case 'bindicus_umd31':
+                    defaultRegion = 'Chr14:1-1000000';
+                    break;
+                case 'bmutus_bosgruv20':
+                    defaultRegion = 'NW_005395160.1:1-1000000';
+                    break;
+                case 'bnapus_branapusv20':
+                    defaultRegion = 'CM002761.2:23000001-24000000';
+                    break;
+                case 'btaurus_arsucd12':
+                    defaultRegion = '6:85000001-86000000';
+                    break;
+                case 'btaurus_umd31':
+                    defaultRegion = '23:28000001-29000000';
+                    break;
+                case 'btaurus_umd311':
+                    defaultRegion = '12:76000001-77000000';
+                    break;
+                case 'cannuum_zunla1ref10':
+                    defaultRegion = 'CM002812.1:144000001-145000000';
+                    break;
+                case 'ccajan_10':
+                    defaultRegion = 'CM003613.1:25000001-26000000';
+                    break;
+                case 'cfamiliaris_31':
+                    defaultRegion = '8:73000001-74000000';
+                    break;
+                case 'chircus_10':
+                    defaultRegion = '19:2000001-3000000';
+                    break;
+                case 'chircus_ars1':
+                    defaultRegion = 'NC_030813.1:86000001-87000000';
+                    break;
+                case 'cjacchus_32':
+                    defaultRegion = '21:3000001-4000000';
+                    break;
+                case 'cquilicii_ccap21':
+                    defaultRegion = 'NW_019376285.1:1000001-2000000';
+                    break;
+                case 'csabaeus_chlsab11':
+                    defaultRegion = 'CAE17:39000001-40000000';
+                    break;
+                case 'csativa_asm186575v1':
+                    defaultRegion = 'Cannabis.v1_scf1_q:1-1000000';
+                    break;
+                case 'csativus_v3':
+                    defaultRegion = 'chr3:16000001-17000000';
+                    break;
+                case 'ddiscoideum_dicty27':
+                    defaultRegion = 'CM000154.2:3000001-4000000';
+                    break;
+                case 'dlabrax_seabassv10':
+                    defaultRegion = 'HG916839.1:21000001-22000000';
+                    break;
+                case 'dpipra_asm171598v1':
+                    defaultRegion = 'MCBO01000495.1:3000001-4000000';
+                    break;
+                case 'drerio_grcz11':
+                    defaultRegion = '13:8000001-9000000';
+                    break;
+                case 'ecaballus_20':
```

Comment:
Should we change the default region for `ecaballus_20` [here](https://github.com/EBIvariation/eva-web/blob/bd78362abeb50abc7bdff6080887bd950cb0bd76/src/js/variant-widget/eva-variant-widget-panel.js#L35) as well, just to be consistent?